### PR TITLE
Add surname to registration form

### PR DESCRIPTION
**Pull request by thomshutt**
At the request of Marketing, this PR adds an explicit surname field to the registration form.

### CODE REVIEW

**Comment by 0xcadams** [comment]
I think we'll actually need to do something with this value here. Where do they want it to go? Hubspot?

**Comment by thomshutt** [comment]
@0xcadams Yep, Hubspot - from the last time I used this page, I thought all values from the form got passed through to Hubspot by default

File: packages/www/components/Register/index.tsx [hunk 1]
```diff
@@ -123,6 +124,20 @@ const Register = ({ id, buttonText, onSubmit, loading, errors }) => {
             onChange={(e) => setFirstName(e.target.value)}
           />
 
+          <TextField
```
Comment (victorges) [inline packages/www/components/Register/index.tsx:128]: Does this already go magically on the request sent to backend? I think it's called `lastName` on the API https://github.com/livepeer/livepeer-com/blob/4e24f8e656e204089b0d0057428820045c162c89/packages/api/src/controllers/user.ts#L314

**Review by victorges — APPROVED** [review]

File: packages/www/components/Register/index.tsx (hunk 1 above)
Comment (thomshutt) [reply packages/www/components/Register/index.tsx:128]: @victorges good call, you were right - I updated it (it's `lastName` in Hubspot too) and tested locally the the last name makes it into the DB _and_ Hubspot

**Pull request merged**